Sources/ArcGISToolkit/Components/Augmented Reality/ARCoachingOverlay.swift

```diff
@@ -45,7 +45,7 @@ struct ARCoachingOverlay: UIViewRepresentable {
     func makeUIView(context: Context) -> ARCoachingOverlayView {
         let view = ARCoachingOverlayView()
         view.delegate = context.coordinator
-        view.activatesAutomatically = false
+        view.activatesAutomatically = true
```

Comment:
The default value for `activatesAutomatically` is `true`. However, I feel it would be beneficial to explicitly set it to `true` here so it is easier to reason its behavior.